cmd/cli/namespace_test.go

```diff
@@ -873,26 +873,28 @@ var _ = Describe("Running the namespace ignore command", func() {
 
 func createNamespaceSpec(namespace, meshName string, enableSideCarInjection bool, ignoreNamespace bool, enableMetrics bool) *v1.Namespace {
 	labelMap := make(map[string]string)
+	annotationMap := make(map[string]string)
 	if meshName != "" {
 		labelMap[constants.OSMKubeResourceMonitorAnnotation] = meshName
 	}
 	ns := &v1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   namespace,
-			Labels: labelMap,
+			Name:        namespace,
+			Labels:      labelMap,
+			Annotations: annotationMap,
```

Comment:
nit (not blocking): I see you're following the pattern that was set for the label map, but for both labels and annotation this feels a bit more idiomatic (removing a variable cleans up readability).

```go
Annotations: make(map[string]string),
....

ns.Annotations[constants.SidecarInjectionAnnotation] = "enabled
```